build.assets/Dockerfile

```diff
@@ -85,18 +85,18 @@ RUN (groupadd ci --gid=$GID -o && useradd ci --uid=$UID --gid=$GID --create-home
      PATH=/usr/local/cargo/bin:$PATH \
      RUST_VERSION=1.56.1
 
- COPY --from=rust:1.56.1 /usr/local/rustup /usr/local/rustup
- COPY --from=rust:1.56.1 /usr/local/cargo /usr/local/cargo
- RUN set -eux \
-     rustup component add rustfmt clippy; \
-     chmod -R a+w $RUSTUP_HOME $CARGO_HOME; \
-     rustup --version; \
-     cargo --version; \
-     rustup target add i686-unknown-linux-gnu; \
-     rustup target add arm-unknown-linux-gnueabihf; \
-     rustup target add aarch64-unknown-linux-gnu; \
-     rustup target list | grep installed; \
-     rustc --version;
+COPY --from=rust:1.56.1 /usr/local/rustup /usr/local/rustup
+COPY --from=rust:1.56.1 /usr/local/cargo /usr/local/cargo
+RUN set -eux \
+    rustup --version; \
+    cargo --version; \
+    rustup component add --toolchain 1.56.1-x86_64-unknown-linux-gnu rustfmt clippy; \
```

Comment:
Do we need this `--toolchain` flag?

Reply:
Unfortunately it looks like so. They didn't re-release cargo with the 1.56.1 patch release and without it, it gets confused and installs clippy and rustfmt for the wrong toolchains which makes them fail during the linting stage. There isn't a prebuilt Rust image from the official sources with those components either.

Reply:
context: removing the toolchain file causes clippy to stop working on the patch release

Reply:
https://drone.teleport.dev/gravitational/teleport/7561